lib/Net/Stripe.pm

```diff
@@ -91,6 +94,7 @@ has 'debug_network' => (is => 'rw', isa => 'Bool',   default    => 0, documentat
 has 'api_key'       => (is => 'ro', isa => 'Str',    required   => 1, documentation => "You get this from your Stripe Account settings");
 has 'api_base'      => (is => 'ro', isa => 'Str',    lazy_build => 1, documentation => "This is the base part of the URL for every request made");
 has 'ua'            => (is => 'ro', isa => 'Object', lazy_build => 1, documentation => "The LWP::UserAgent that is used for requests");
+has 'api_version'   => (is => 'ro', isa => 'Net::Stripe::Types::api_version', documentation => "This is the value of the Stripe-Version header you wish to use for API calls");
```

Comment:
I'd make this one lazy, and default to `_get_account_api_version` (A)

Reply:
my instinct here was to not transmit the API version unless explicitly set. retrieving the API version only to send it back explicitly, rather that relying on Stripe's default behavior, feels like it provides opportunity for a disconnect if things should change on the Stripe side. but this is mostly from my gut, and not based on any concrete technical scenario.

Reply:
Sorry - I'm a bit confused. I thought this was only for checking that the user was using the right version of `Net::Stripe` compared with the default version in their account, rather than telling Stripe's API which version the request is speaking in.

Reply:
not sure if my inline comment will register with this conversation, but we send the Stripe-Version header on 1554 of this commit